### PR TITLE
Bump

### PR DESCRIPTION
**Pull request by ocefpaf**
@duncanwp hopefully this one will work as expected.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Yep. No double CIs! Sorry for the confusion in #11 not sure why that happened. I usually do a fork -> branch -> PR from CLI. Did you do something different?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -57,4 +54,4 @@ about:
 
 extra:
   recipe-maintainers:
-     - duncanwp
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I also rendered and cleaned some white spaces. Nothing really important though.

**Comment by duncanwp** [comment]
Great, thanks. 

Since I had a branch on here, I did a fork, fetch, checkout and push. I didn't think it should make a difference. Perhaps it was your rerender that did it?

File: recipe/meta.yaml (hunk 1 above)
Comment (duncanwp) [inline recipe/meta.yaml]: Yes, I spotted, thanks.

**Comment by ocefpaf** [comment]
> Perhaps it was your rerender that did it?

Probably not as the re-render should not affect how github behaves. It is only another "change" to commit.

> Since I had a branch on here, I did a fork, fetch, checkout and push.

you can try `git remote -v` and check if the fork `origin` and `upstream` are correct.

Feel free to merge this when the CIs passes. Anything you need ping me.

**Pull request merged**